package.json

```diff
@@ -589,7 +589,7 @@
           "description": "If enabled, you can manually input the model name in the model id in the select dropdown. This is required for some APIs, like ollama.",
           "order": 49
         },
-        "chatgpt.apiVersion": {
+        "chatgpt.azureApiVersion": {
```

Comment:
So I checked the [VS Code spec](https://code.visualstudio.com/api/references/contribution-points#Configuration-schema), this is indeed the only way to change the title.

Changing the key is okay, but if we rename a setting's key, we _must_ gracefully move users over to the new config setting in our startup script.

Reply:
Totally agree 👍